modules/network-config-cr.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-network-operator.adoc
+[id="network-default-cr-{context}"]
+= Cluster network configuration custom resource
+
+The basic configuration for the `Network Operator` is set in the `Network`
```

Comment:
A bit of terminology: the Kubernetes object is "Network.config.openshift.io". "Nework" is the object name. "config.openshift.io" is the group. These happen to be defined as CRDs but that's not important for end-users to know.

So I would word this as something like:

The cluster's network configuration is set in the "Network.config.openshift.io" object. All fields in the "Spec" subresource are mandatory.

The object is automatically created by the openshift-installer.